packages/mui/test/__snapshots__/Form.test.tsx.snap

```diff
@@ -14176,7 +14176,8 @@ label[data-shrink=false]+.MuiInputBase-formControl .emotion-3:focus::-ms-input-p
                 "height": 0,
                 "left": 0,
                 "overflow": "hidden",
-                "padding": 0,
+                "paddingBottom": 0,
+                "paddingTop": 0,
```

Comment:
this is because not all package-lock files were used, and effectively an update for some of the dependencies in repository (change is coming from the new mui version)